app/views/my_facilities/bp_controlled.html.erb

```diff
@@ -1,122 +1,167 @@
 <div class="row">
   <div class="col-lg-12">
-    <div class="card card-responsive">
-      <div class="d-flex align-baseline jc-between">
-        <h3>
-          BP controlled
-        </h3>
-        <p class="c-grey-dark">
-          Last updated: <%= @last_updated_at %>
+    <div class="card-responsive bg-white bs-small br-4px">
+      <div class="pt-16px ph-16px">
+        <div class="d-flex align-baseline jc-between">
+          <h3>
+            BP controlled
+          </h3>
+          <p class="c-grey-dark">
+            Last updated: <%= @last_updated_at %>
+          </p>
+        </div>
+        <p class="mb-4px c-grey-dark">
+          <strong>What to look for:</strong> High percentages are good, and 70% or higher is ideal 
+        </p>
+        <p class="c-grey-dark c-print-black">
+          <strong>Numerator:</strong> <%= t("numerator_copy.bp_controlled") %>, <strong>Denominator:</strong> <%= t("denominator_copy") %>
         </p>
       </div>
-      <p class="mb-4px c-grey-dark">
-        <strong>What to look for:</strong> High percentages are good, and 70% or higher is ideal 
-      </p>
-      <p class="c-grey-dark c-print-black">
-        <strong>Numerator:</strong> <%= t("numerator_copy.bp_controlled") %>, <strong>Denominator:</strong> <%= t("denominator_copy") %>
-      </p>
-      <div class="d-flex fw-wrap mb-16px">
+      <div class="d-flex fw-wrap mb-16px ph-16px">
         <%= render "shared/my_facilities_filters" %>
       </div>
-      <% if @facility_sizes.empty? %>
-        <%= render "shared/my_facilities_table_empty_state" %>
-      <% end %>
-      <% @display_sizes.each do |size| %>
```

Comment:
I believe changing this from display_sizes to facility_sizes has created a bug. If you select a size from the right-most dropdown, you see all sizes in the facility group.